mlrun/datastore/v3io.py

```diff
@@ -94,13 +93,15 @@ def get_storage_options(self):
         )
         return self._sanitize_storage_options(res)
 
-    def _upload(self, key: str, src_path: str, max_chunk_size: int = ONE_GB):
+    def _upload(
+        self, key: str, src_path: str, max_chunk_size: int = V3IO_UPLOAD_CHUNK_SIZE
+    ):
         """helper function for upload method, allows for controlling max_chunk_size in testing"""
         container, path = split_path(self._join(key))
         file_size = os.path.getsize(src_path)  # in bytes
-        if file_size <= ONE_MB:
+        if file_size <= V3IO_UPLOAD_CHUNK_SIZE:
```

Comment:
Like I said all under 'if file_size <= V3IO_UPLOAD_CHUNK_SIZE' should be removed